src/storage/resourceManager.ts

```diff
@@ -38,12 +40,6 @@ export type TopicsByKafkaCluster = Map<string, KafkaTopic[]>;
 /** Type for storing {@link Schema}s in extension state, where the parent {@link CCloudSchemaRegistry} ID is the key. */
 export type CCloudSchemaBySchemaRegistry = Map<string, Schema[]>;
 
-/** Single URI's confluent-extension-centric metadata */
-export type UriMetadata = Map<UriMetadataKeys, string>;
-
-/** Map of string of uri for a file -> dict of its confluent-extension-centric metadata */
-export type AllUriMetadata = Map<string, UriMetadata>;
-
```

Comment:
Unused before this PR, respelled and moved over to https://github.com/confluentinc/vscode/blob/ed745ba707a1760003421f88773844e26d857b9b/src/storage/types.ts